articles/dms/tutorial-sql-server-managed-instance-online.md

```diff
@@ -70,6 +70,8 @@ To complete this tutorial, you need to:
     > Regarding the storage account used as part of the migration, you must either:
     > * Choose to allow all network to access the storage account.
     > * Turn on [subnet delegation](../virtual-network/manage-subnet-delegation.md) on MI subnet and update the Storage Account firewall rules to allow this subnet.
+    
+     >- Azure Storage account with the private endpoint is not supported by Azure Database Migration Service.
```

Comment:
```suggestion
     >- Azure Storage account with the private endpoint is not supported by Azure Database Migration Service.
```
 > - Azure Storage account with the private endpoint is not supported by Azure Database Migration Service.